app/components/brandlogo.js

```diff
@@ -0,0 +1,28 @@
+import dynamic from "next/dynamic";
```

Comment:
Please do not use dynamic or inview.   These are pure unnecessary baggages.

Just using `<Image>` will get you all the benefits as stated in its docs.  Please read the doc.

Reply:
I understand, I will remove dynamic and inview and use the properties of `<Image>` for optimisation